copy_input_files.sh

```diff
@@ -11,9 +11,10 @@ FILES_INFO=(
   pre/load/current/index.status:1500000
 )
 
-# Delete old input files
+# zip/delete old input files
 if [ -d ./logs/input ]; then
-    find ./logs/input/ -name "input.20*-*-*_*-*-*" -type d -mtime +7 -exec rm -rf '{}' \;
+    find ./logs/input/ -name "input.20*-*-*_*-*-*" -type d -mtime +3 -exec tar cvzf '{}'.tar.gz '{}' \; -exec rm -r '{}' \;
+    find ./logs/input/ -name "input.20*-*-*_*-*-*" -mtime +14 -exec rm -rf '{}' \;
```

Comment:
Maybe let's not increase mtime here until we don't get a better sense of the space that we will end up using? We could keep the 7 days limit for now.